src/main/java/io/kestra/plugin/airbyte/connections/Sync.java

```diff
@@ -127,5 +148,10 @@ public static class Output implements io.kestra.core.models.tasks.Output {
             title = "The jobId created"
         )
         private final Long jobId;
+
+        @Schema(
+                title = "Whether a sync was already running or not"
+        )
+        private final Boolean alreadyRunning;
```

Comment:
I wonder if communicating that a sync is already running has some usage? Would it be enought to log a warning?

Reply:
Yes it has, in my case I don't want to perform same execution flow if the sync is already running or not 🤔